### ISSUE TITLE
The Nearby Gates button should be a radio button within the TSP options

### ISSUE THREAD

**Issue by MeemawHustlin** [MichaelEbert/OblivionProgressTracker#267]
Since nearby gates relates to pathing, it makes more sense for it to be toggled within the list of path options to keep a clear distinction between map icons that can toggled vs paths connecting those icons.

The only way we would want it otherwise would be if there is a use-case for seeing the nearby gates while following one of the other TSPs, but I don't foresee that being practical within route logic.

I tried figuring out how to fix it on my own but it looks like nearby gates are implemented differently than the TSPs or something, so I can't get the nearby gates radio button to clear the previous path selection, it just overlays nearby gates on top of the previously selected TSP despite the radio button unchecking for the other options.

**Comment by MichaelEbert** [comment]
Yeah, nearby gates are currently implemented as a separate layer. This allows for only some gates to be shown, instead of all gates and all locations. I probably won't fix it myself before v4, but if you're able to do it go for it.

You can play around with the map layers in the web console by using
`mapjs.getOverlay()`
so you could enable drawing of the nirnroot layer by saying:
`mapjs.getOverlay().layers.get("nirnroots").visible=true`
and then move your mouse over the map to have it redraw.

You can also force redraw any component by calling anything with a draw method. such as:
`mapjs.getOverlay().layers.get("nirnroots").draw(mapjs.getCtx())
or 
`mapjs.getOverlay().draw(mapjs.getCtx())

but be careful as it'll redraw on its own as soon as your mouse enters the map portion of the page.
